libcontainerd/client_windows.go

```diff
@@ -157,6 +158,10 @@ func (clnt *client) Create(containerID string, spec Spec, options ...CreateOptio
 		}
 	}
 
+	if spec.Windows.Servicing {
+		cu.ContainerType = "Servicing"
```

Comment:
If using a string, should be a constant. Personally I would prefer a single boolean property to indicate that it's explicitly for servicing. But I know that changes the interface to HCS.